src/buffer/input_view/completion.rs

```diff
@@ -1056,16 +1176,38 @@ impl Command {
 #[derive(Debug, Clone)]
 struct Arg {
     text: &'static str,
-    optional: bool,
+    kind: ArgKind,
     tooltip: Option<String>,
 }
 
+#[derive(Debug, Clone)]
+enum ArgKind {
```

Comment:
A very very minior nit would be to make this enum name a little more descriptive to what it relates to.